src/dotnet/MsbuildProject.cs

```diff
@@ -16,44 +19,49 @@ internal class MsbuildProject
     {
         const string ProjectItemElementType = "ProjectReference";
 
-        public ProjectRootElement Project { get; private set; }
+        public ProjectRootElement ProjectRoot { get; private set; }
```

Comment:
Can we call this ProjectRootElement? I'd expect ProjectRoot to be a DirectoryInfo. Of course, this is an improvement on what was here before :)